log/wrapper.go

```diff
@@ -113,10 +113,11 @@ func (w Wrapper) ToThriftLogger() thrift.Logger {
 // - "std": StdWrapper with default stdlib logger
 // (log.New(os.Stderr, "", log.LstdFlags)).
 //
-// - "zap": ZapWrapper on default level (Info).
+// - "zap": ZapWrapper on default level (Info) with no kv pairs.
 //
-// - "zap:level": ZapWrapper with given level, for example "zap:error" means
-// ZapWrapper on Error level.
+// - "zap:level:k1:v1:k2:v2...": ZapWrapper with given level and kv pairs, for
```

Comment:
why not

```suggestion
// - "zap:level:k1=v1:k2=v2...": ZapWrapper with given level and kv pairs, for
```

Reply:
Also, I think I would consider having the keyvals be `,`-separated so that you don't preclude yourself from adding additional (optional) configuration after them in the future.  (you can't add arguments after a variadic)

e.g.

```
zap:level:k1=1,k2=2,k3=3:console=true
```

or something (my imagination about what we might want to configure is failing me right now, but you get the idea)

Reply:
Oh, also, I would make it explicit that this is still optional by explaiing it separately from level.

```
zap:level
 - docs
zap:level:k1=1,k2=2
 - docs
```

or w/e.

Also, I think this is a non-breaking change as long as nobody uses :s in their level name, which seems probable?

Reply:
I think this approach would still have the problem of what if the user does not want to add any kv pairs, but they do want to add `console=true` (in the hypothesis scenario that we add it in the future), so we need to choose between allowing `zap:level:console=true` and `zap:level::console=true` (e.g they must have an explicit empty "kv block").

Reply:
What is the semantic of `console=true` here? I think I am lacking context on this.

Reply:
I was making up a hypothetical future thing we might want to configure, sorry.  It's just meant to be "anything that we might want to add in the future"

Reply:
I am not advocating against using `=` I'm advocating against using `:` to separate keyval pairs. 
 `zap:level:k1=1,k2=2:somethingelse` is more readable than `zap:level:k1=1:k2=2!!!somethingelse` or whatever we would have to do to work around the fact that we allowed arbitrary number of `:`-separate keyval pairs when we need to extended this in the future.
 
 And I say "when" because if we are having to extend this once, it's very likely we will need to extend it again.  So we should plan for that.